lib/ruby_lsp/parameter_scope.rb

```diff
@@ -1,6 +1,8 @@
 # typed: strict
 # frozen_string_literal: true
 
+require "set"
```

Comment:
Let's please move this to `internal.rb` [below this line](https://github.com/Shopify/ruby-lsp/blob/dd3a94a665591de60b17748ecf53b90368b74e5d/lib/ruby_lsp/internal.rb#L9).

Reply:
Done!